src/redux/delete-reservation/deleteReservationSlice.js

```diff
@@ -0,0 +1,65 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const url = 'http://localhost:4000';
+
+export const fetchReservations = createAsyncThunk('bookings/fetchReservations', async (_, { rejectWithValue }) => {
+  try {
+    const response = await axios.get(`${url}/bookings`, { withCredentials: true });
+    return response.data;
+  } catch (error) {
+    return rejectWithValue(error.response.data);
+  }
+});
```

Comment:
- [ ] I'd like to say that I've done well implementing the async thunk action to fetch the data, but please make sure the reservations are brought from the back-end updating the initial state properly. Currently, I can't see any updated data.

`Hints`: Maybe you can add the necessary headers, especially the `Authorization` token.